### DISCUSSION TITLE
Cannot pip install pybag on macOS 10.15.7 (19H2026)

### DISCUSSION THREAD

**Discussion by loyc12** [pygame-web/pygbag#132] [Help zone] [answered]
Hi, I am trying to use pybag to compile a python/pygame project in wasm (duh) for a school project, but I am encountering an issue ( that may be due to my school's macOS version : **10.15.7 (19H2026)** ).

When running either _' python3 -m pip install --user pybag '_ or _' pip3 install --user pybag '_, I receive error messages mentioning a conflict:

_( I also have this when not using --user, but I need to since I do not have the permissions to pip install on root )_

```
ERROR: Cannot install [...] pybag==2.2.8 because these package versions have conflicting dependencies.

The conflict is caused by:
    pybag 2.2.8 depends on pywin32>=301
    [...]
```

However, since I am on macOS, I do not have and cannnot install pywin32, as it is solely for windows. From my searches online, it seems that pybag is supposed to work on both mac and linux as well as windows, so I do not understand why it requires pywin32 for me.

Again, I suspect this might be due to the older macOS version my school uses.. somehow(?). Any help of conrfirmation I am not the only one to have this problem is welcomed.

Thanks!

**Comment by loyc12** [comment, marked as answer]
Nevermind I am a blind idiot, I forgot the G in pygbag 💀

**Comment by pmp-p** [comment]
> Any help of conrfirmation I am not the only one to have this problem is welcomed.

indeed, don't bash yourself it's a classic :)

**Comment by Fireblaster340** [comment]
Same here, Ig we both lack braincells
